packages/controller/lib/setup/setupBackup.js

```diff
@@ -629,6 +629,9 @@ class BackupRestore {
     }
 
     _copyBackupedFiles(backupDir) {
+        if (!fs.existsSync(backupDir)) {
```

Comment:
The problem of https://github.com/ioBroker/ioBroker.js-controller/issues/1768 is not, that no path is passed to the method. The problem is, that the path is non existing!